doc/source/ray-core/doc_code/direct_transport_gloo.py

```diff
@@ -130,7 +131,19 @@ def sum(self, tensor: torch.Tensor):
 assert torch.allclose(*ray.get([sum1, sum2]))
 # __gloo_intra_actor_end__
 
+# Correct example of ray.get().
 # __gloo_get_start__
-print(ray.get(tensor))
+print(ray.get(tensor, _tensor_transport="object_store"))
 # torch.Tensor(...)
 # __gloo_get_end__
+
+# Wrong example of ray.get(). Since the tensor transport in the @ray.method decorator is Gloo,
+# ray.get() will try to use Gloo to fetch the tensor, which is not supported.
+# In this case, users must specify the correct tensor transport explicitly in ray.get().
+with pytest.raises(ValueError) as e:
```

Comment:
using pytest like this seems awfully odd to me.  Why not just raise(ValueError)? Introducing a pytest dep here seems awfully heavy handed.